packages/smart-contracts/src/contracts/lib/RinkebyChainlinkAggregatorCaller.sol

```diff
@@ -0,0 +1,103 @@
+pragma solidity ^0.5.0;
+
+interface AggregatorInterface {
+  function latestAnswer() external view returns (int256);
+
+  function latestTimestamp() external view returns (uint256);
+
+  function latestRound() external view returns (uint256);
+
+  function getAnswer(uint256 roundId) external view returns (int256);
+
+  function getTimestamp(uint256 roundId) external view returns (uint256);
+
+  event AnswerUpdated(int256 indexed current, uint256 indexed roundId, uint256 updatedAt);
+  event NewRound(uint256 indexed roundId, address indexed startedBy, uint256 startedAt);
+}
+
+
+/**
+ * @title ChainlinkAggregatorCaller for Rinkeby
+ */
+contract ChainlinkAggregatorCaller {
```

Comment:
Should be a library? Shouldn't we have one library instantiating values depending on the network?

Reply:
This piece of code is meant to be used only once. Plus, using a library would require two deployments. 
Maybe I am missing something, tho?